styles/auth.css

```diff
@@ -0,0 +1,48 @@
+.signup .input,
+.login .input {
+  height: 100%;
+  border-radius: 2px;
+  outline: none;
+
+  font-size: 1rem;
+}
+
+p.login-link,
+p.signup-link {
+  color: var(--primary-color);
+  font-weight: bold;
+}
+
+.login .card .h--2,
+.signup .card .h--2 {
+  text-align: center;
+}
+
+.signup .card,
+.login .card {
+  width: max-content;
+}
+
+.login .input__container,
+.signup .input__container {
+  width: 100%;
+}
+
+.login .input,
+.signup .input {
+  height: 100%;
+  border-radius: 2px;
+  outline: none;
+  font-size: 1rem;
+}
+
+p.signup-link,
+p.login-link {
+  color: var(--primary-color);
+  font-weight: bold;
+}
+
+.t-and-c {
```

Comment:
what's t-and-c? try to give a more meaningful name.

Reply:
changed it to terms-conditions